package.json

```diff
@@ -12,10 +12,11 @@
     "lint-src-and-fix": "tslint \"src/**/*.ts\" --fix",
     "build": "rimraf dist && npx ttsc && ts-node --transpileOnly scripts/build/removeEmptyFiles",
     "build-declarations": "rimraf dist-declarations && ts-node --max-old-space-size=8192 --transpileOnly scripts/generation/main create-code-block-writer-file create-declaration-file",
-    "test": "cross-env TS_NODE_COMPILER=\"ttypescript\" TS_NODE_TRANSPILE_ONLY=\"true\" mocha --opts mocha.opts --grep @performance --invert",
-    "test-debug": "cross-env TS_NODE_COMPILER=\"ttypescript\" TS_NODE_TRANSPILE_ONLY=\"true\" mocha --opts mocha.opts --grep @performance --invert --inspect-brk",
+    "test": "cross-env TS_NODE_COMPILER=\"ttypescript\" TS_NODE_TRANSPILE_ONLY=\"true\" mocha --opts mocha.opts --grep \"^(?!.*@performance)\"",
+    "test-debug": "cross-env TS_NODE_COMPILER=\"ttypescript\" TS_NODE_TRANSPILE_ONLY=\"true\" mocha --opts mocha.opts --grep \"^(?!.*@performance)\" --inspect-brk",
+    "test-watch": "watch \"npm test\" src",
```

Comment:
There is a mocha `--watch src`. Is using `watch` necessary?

Reply:
It didn't work for me, but I'm not so familiar with mocha, let me research a little bit.

Reply:
Thanks for looking into that!